admin_site/wagtail_hooks.py

```diff
@@ -72,3 +81,5 @@ def register_instance_chooser():
     )
 
 hooks.register('register_admin_menu_item', register_instance_chooser)
+
+register_snippet(DatasetSchemaViewSet)
```

Comment:
I registered this here in `admin_site/wagtail_hooks.py`, but there might be a better suited place. Especially when DVC management commands are fixed, they might have to be placed live in some app, and this might fit in the same app as well...?